src/Grid.js

```diff
@@ -312,4 +338,8 @@ export default class Grid extends CollectionWrapper {
         this._mainSpacing = num;
         this._crossSpacing = num;
     }
+
+    get spacing() {
+        return this._spacing;
+    }
```

Comment:
The `CollectionWrapper` from which `Grid` extends already exposes this _getter_, so it is redundant here.